manifests/package.pp

```diff
@@ -70,18 +70,23 @@
         'rpm':   { $package_provider = 'rpm'   }
         default: { fail("Unknown file extension '${extension}'.") }
       }
+
+      $package_require = undef
```

Comment:
Thanks. We'll soon have fixes and test coverage for this, thanks to you and @mrbanzai.